cmd/nonmem.go

```diff
@@ -49,6 +49,29 @@ const sgeExecutionTemplate string = `#!/bin/bash
 {{ .Command }}
 `
 
+//Parse type 2 refers to evenly load balanced work
+//Transfer Type 1 refers to MPI
+//TIMEOUTI 100 means wait 100 seconds for node to become available
+//TIMEOUT 10 means wait 10 seconds for work to complete -> Should default to 10
```

Comment:
many models can take days or even weeks to run. It would be best to have a default of no timeout (can timeout set to 0 or maybe a negative number do that?)

Reply:
This is primarily from the documentation on Nonmem:

```
; NODES=number of nodes (that is process, whether cores or computers)
; SINGLE node: NODES=1
; MULTI node (node means process, whether cores or computers): NODES>1
; WORKER node: NODES=0
;
; parse_num=number of subjects to give to each node
; parse_type=0, give each node parse_num subjects
; parse_type=1, evenly distribute numbers of subjects among available nodes
; parse_type=2, load balance among nodes
; parse_type=3, assign subjects to nodes based on idranges
; parse_type=4, load balance among nodes, taking into account loading time.
; This setting of parse_type will assess ideal number of nodes.
; If loading time too costly, will eventually revert to single CPU mode. ;
; timeouti=seconds to wait for node to start. if not started in time,
; deassign node, and give its load to next worker, until next iteration
; timeout=minutes to wait for node to compelte. if not completed by then, ; deassign node, and have manager complete it.
; paraprint=1 print to console the parallel computing process. Can be
; modified at run-time with ctrl-B toggle.
; Regardless of paraprint setting, <control_stream>.log always records
; parallelization progress.
```
I need to try setting it to 0 and seeing what it does  because the nonmem docs don't specifically say anything about unlimited timeout in parallel execution mode

Reply:
So the job still finishes with a defined timeout of 0. Not sure if it's exerting extraneous pressure on the manager mentioned above, but if it works fine we can basically always set this to 0 and remove the parameter altogether

Reply:
The manager according to those docs is the thread invoking the nonmem command. So from what I understand, if the parallel node doesn't finish the work in the time frame specified, it's handed back to the manager to do the work. So to some degree it ceases to be parallel.

> The manager process is the user’s process that runs the nmfe74 script, reads the control stream file, executes NMTRAN, and runs the main NONMEM process. The worker process is NONMEM in worker mode, not taking any input from the user, only from the manager NONMEM process.

Reply:
Also using a negative number breaks nonmem compilation:
```
 * THE MAXIMUM NUMBER OF WARNINGS OF ONE OR MORE TYPES WAS REACHED.
 IT IS POSSIBLE THAT SOME WARNING MESSAGES WERE SUPPRESSED.
Recompiling certain components
  TIMEOUT MUST BE INTEGER BETWEEN (0,2147483647)
```

2147483647 looks like a sane value to use. I mean that's like 4000 years